roles/lib_utils/action_plugins/sanity_checks.py

```diff
@@ -363,3 +444,34 @@ def run(self, tmp=None, task_vars=None):
         result["msg"] = "Sanity Checks passed"
 
         return result
+
+
+def is_registry_match(item, pattern):
+    """returns True if the registry matches the given whitelist pattern
+
+    Unlike in OpenShift, the comparison is done solely on hostname part
+    (excluding the port part) since the latter is much more difficult due to
+    vague definition of port defaulting based on insecure flag. Moreover, most
+    of the registries will be listed without the port and insecure flag.
+    """
+    item = "schema://" + item.lstrip("http://").lstrip("https://")
```

Comment:
Use `.replace()` to remove multi character strings.

Reply:
Good catch. Changed to `.split('://',1)[-1]` instead.